client/influxdb.go

```diff
@@ -624,6 +628,17 @@ func (c *Client) Addr() string {
 	return c.url.String()
 }
 
+// checkPointTypes ensures no unsupported types are submitted to influxdb, returning error if they are found.
+func checkPointTypes(p Point) error {
+	for _, v := range p.Fields {
+		switch v.(type) {
+		case uint64:
```

Comment:
`int64` is the data type on the server but everything ultimately get marshaled to a string representation via the line protocol.  When it's parsed and has a trailing `i`, it's parsed into an `int64` regardless of the client integer type.

I think it's fine to reject here.  Should be easy enough to allow them in the future if this causes a problem.